src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeCloner.cs

```diff
@@ -107,10 +118,21 @@ private BindingDataResolver GetResolver(PropertyInfo propInfo, INameResolver nam
                 return GetAppSettingResolver((string)originalValue, appSettingAttr, nameResolver, propInfo);
             }
             // try to resolve with auto resolve ({...}, %...%)
-            if (autoResolveAttr != null && originalValue != null)
+            if (autoResolveAttr != null)
             {
-                _autoResolves[propInfo] = autoResolveAttr;
-                return GetTemplateResolver((string)originalValue, autoResolveAttr, nameResolver, propInfo, contract);
+                if (autoResolveAttr.Default == DefaultResolveValue.MemberName)
+                {
+                    if (originalValue == null || string.IsNullOrWhiteSpace(originalValue.ToString()))
```

Comment:
This was tricky since originalValue isn't a string. 
That actually is directly related to:  
[https://github.com/Azure/azure-webjobs-sdk/issues/1110](https://github.com/Azure/azure-webjobs-sdk/issues/1110) 

I'll just fix them both. 

---
In reply to: [113585460](https://github.com/Azure/azure-webjobs-sdk/pull/1124#discussion_r113585460) [](ancestors = 113585460)